source/AMDSClient.cpp

```diff
@@ -199,7 +214,7 @@ void AMDSClient::onSocketDataReady(AMDSClientTCPSocket* clientTCPSocket, AMDSCli
 			if (introspectionRequest) {
 				if (introspectionRequest->readReady()) {
 					const QStandardItemModel* model = qobject_cast<const QStandardItemModel*>(requestType->model());
-					for(int x = 1; x < 7; x++){
+					for(int x = 1; x < AMDSClientRequestDefinitions::InvalidRequest; x++){
```

Comment:
I'm a little wary of iterating through enums in this fashion. If someone adds another value to the enum after InvalidRequest, then it won't be included in this loop anywhere. I'm imagining the client stuff will change a lot, so this might not even be relevant. Might just be worth adding a documenting note to AMDSClientRequestDefinitions::RequestType that InvalidRequest must be the last value.